.github/tests/images.json

```diff
@@ -0,0 +1,46 @@
+{
+  "spire-server/values.yaml": [
+    {
+      "query": "tests.bash.image",
+      "filter": "^latest-",
+      "sort-flags": []
+    }
+  ],
+  "spire-agent/values.yaml": [
+    {
+      "query": "waitForIt.image",
+      "filter": "^latest-",
+      "sort-flags": []
+    }
+  ],
+  "spiffe-csi-driver/values.yaml": [
+    {
+      "query": "nodeDriverRegistrar.image",
+      "filter": "^v",
+      "sort-flags": []
+    }
+  ],
+  "spiffe-oidc-discovery-provider/values.yaml": [
+    {
+      "query": "insecureScheme.nginx.image",
+      "filter": "^[0-9]\\+\\.[0-9]\\+\\.[0-9]\\+-alpine$",
+      "sort-flags": []
+    }, {
+      "query": "telemetry.prometheus.nginxExporter.image",
+      "filter": "^[0-9]\\+\\.[0-9]\\+\\.[0-9]\\+$",
+      "sort-flags": ["-t", ".", "-k1,1n", "-k2,2n", "-k3,3n"]
+    },
+    {
+      "query": "tests.bash.image",
+      "filter": "^latest-",
```

Comment:
We switched to use the semver tag here. So this also abstracts away the need of daily patching.

Reply:
can switch to semver for bash. waitForIt has no non latest tags.

Reply:
Have a look at https://github.com/spiffe/helm-charts/pull/356#pullrequestreview-1484828752
I think we probably want to stick with latest variants because they are immutable.

Reply:
Oh found the answer further down in this thread.

https://github.com/spiffe/helm-charts/pull/249#issuecomment-1674018842